keyboards/splitkb/zima/rules.mk

```diff
@@ -0,0 +1,28 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = no      # Virtual DIP switch configuration
```

Comment:
```suggestion
BOOTMAGIC_ENABLE = lite    # Virtual DIP switch configuration
```

Reply:
What is the reason for wanting to add this to the default layout?

Reply:
having bootmagic lite is a nice fallback.

Reply:
> What is the reason for wanting to add this to the default layout?

Users tend to forget having a `RESET` key. I like to recommend having bootmagic lite enabled so they don't have to disassemble their board.

> But firmware space.

I see, alrighty then.